### ISSUE TITLE
Assignment 11 Status

### ISSUE THREAD

**Issue by jeehoonkang** [snu-sf/pl2015#136]
Hi all,
- Assignment 10 is issued. Please see the [README](README.md) for more details.
- Please read [Homework.md](Homework.md#how-to-submit) to know how to fetch homework and submit your answer.
  - You have to run `./fetch-homework.sh` to get assignment 11.
  - Do your homework by editing `sf/Assignment11_??.v`.
  - Make sure `make` works without errors.
  - Commit and push to your GitHub repository.
  - Make sure `https://github.com/$YOURID/pl2015` contains the change you made.
- Stay tuned for [README.md](README.md) and this issue on the assignment.
- You are NOT allowed to use the [admit] tactic.
- You are ALLOWED to use any tactics including: [tauto], [intuition], [firstorder], [omega].
- Just leave [exact FILL_IN_HERE] for those problems that you fail to prove.
- Do NOT add any additional `Require Import/Export`.

Sincerely,
Jeehoon

**Comment by alkaza** [comment]
Please consider changing the due date of HW11 to June 11th

**Comment by jeehoonkang** [comment]
- We delayed the due date of assignment 11 to 11th June. Please see the README.
- Assignment 12 will be assigned around 4th June, and it's due 18th June (just before the final).
- Assignment 13 will be assigned around 11th June, and it's due 18th June (just before the final).

Jeehoon

**Comment by jaewooklee93** [comment]
Is final decided?

**Comment by alkaza** [comment]
Some people in this class have Algorithms exam on the 18th of June...

**Comment by jeehoonkang** [comment]
@jaewooklee93 @alkaza We will gather polls til the next Thursday, and decide at the class.

Jeehoon

**Comment by AdamBJ** [comment]
Will the grader be released soon? Also, @jeehoonkang did you have a chance to talk to Professor Hur about the grading issue I told you about?

**Comment by jeehoonkang** [comment]
Here is the grader: http://sf.snu.ac.kr/jeehoon.kang/pl2015/Assignment11_grader.zip

**Comment by jeehoonkang** [comment]
@AdamBJ Yes I just asked Professor Hur, and the short answer is: no I cannot. Sorry.

**Comment by alkaza** [comment]
But for this assignment the delay due was the same as for Assignment 12, were the submissions collected?

**Comment by jeehoonkang** [comment]
Sorry I omitted message: delay submissions are collected.
